.circleci/config.yml

```
@@ -154,6 +154,22 @@ commands:
       - run:
           name: Post comment on GitHub
           command: ./scripts/notify-github.sh "<< parameters.data >>"
+  pack_and_build_sdk:
```

I feel like the order of `build` then `pack` makes more sense for the name, what do you think?
```suggestion
  build_and_pack_sdk:
```